src/services/UnitsService.js

```diff
@@ -0,0 +1,81 @@
+import { $injector } from '../injection';
+
+const KILOMETER_IN_METERS = 1000;
+const SQUAREDKILOMETER_IN_SQUAREDMETERS = 1000000;
+const HEKTAR_IN_SQUAREDMETERS = 10000;
+const PROVIDER_METRIC = {
```

Comment:
Let's name "private"  static properties in a Pascal_Snake_Case  manner: `Kilometers_In_Meters`.
UPPER_SNAKE_CASE should be reserved for "public" static fields